scripts/binary/build.coffee

```diff
@@ -229,18 +296,84 @@ buildCypressApp = (platform, version, options = {}) ->
     # hint: you can see all symlinks in the build folder
     # using "find build/darwin/Cypress.app/ -type l -ls"
     console.log("platform", platform)
-    electronDistFolder = meta.buildAppDir(platform, "packages", "electron", "dist")
+    electronDistFolder = distDir("packages", "electron", "dist")
     la(check.unemptyString(electronDistFolder),
       "empty electron dist folder for platform", platform)
 
     console.log("Removing unnecessary folder '#{electronDistFolder}'")
-    fs.removeAsync(electronDistFolder).catch(_.noop)
+    fs.removeAsync(electronDistFolder) # .catch(_.noop) why are we ignoring an error here?!
+
+  removeDuplicateElectron = ->
+    log('#removeDuplicateElectron')
+    if platform != "linux"
+      console.log("no duplicate file on platform #{platform}")
+      return Promise.resolve()
+    # for some reason I see duplicate "electron" binary left in the build folder
+    # build/linux-unpacked/Cypress <-- good built app
+    # build/linux-unpacked/electron <-- same file as "Cypress" app
+    CypressFilename = buildDir("Cypress")
+    cypressFilename = buildDir("cypress")
```

Comment:
you mention previously that `Cypress` and `electron` are the duplicates... but your code is checking for `Cypress` and `cypress`